plugins/css-has-pseudo/src/browser.js

```diff
@@ -4,18 +4,9 @@ import '@mrhenry/core-web/modules/~element-qsa-has.js';
 import extractEncodedSelectors from './encode/extract.mjs';
 import encodeCSS from './encode/encode.mjs';
 
-function hasNativeSupport(document) {
+function hasNativeSupport() {
 	try {
-		// Chrome does not support forgiving selector lists in :has()
-		document.querySelector(':has(*, :does-not-exist, > *)');
-		document.querySelector(':has(:has(any))');
-
-		// Safari incorrectly returns the html element with this query
-		if (document.querySelector(':has(:scope *)')) {
-			return false;
-		}
-
```

Comment:
The specification changed so we need to update the feature detection to match.
Functional changes were done in the `querySelector(':has()')` polyfill from `core-web`